CMakeLists.txt

```diff
@@ -80,11 +80,19 @@ message(STATUS "Current git version ${CURRENT_GIT_VERSION}")
 ################################################################################
 # Version information
 ################################################################################
-
-set(FDB_PACKAGE_NAME "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}")
-set(FDB_VERSION ${PROJECT_VERSION})
+if (FDB_RELEASE_CANDIDATE)
+    set(FDB_VERSION ${PROJECT_VERSION}-rc1)
```

Comment:
Is it possible to have multiple "rc"s? If so, we might need a `RELEASE_CANDIDATE_VERSION`.

Reply:
My thought is that it is a string. It would need to be modified the same as any other version string (if it is being changed), why introduce another variable to hold the string that is being modified?

Reply:
I've added a variable that can be overridden from the command line, so that a new RC would not necessarily require a modification of `CMakeLists.txt`.